media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -5334,12 +5334,21 @@ DdiMedia_QueryVideoProcPipelineCaps(
     }
     else if ((context & DDI_MEDIA_MASK_VACONTEXT_TYPE) == DDI_MEDIA_VACONTEXTID_OFFSET_VP)
     {
-        pipeline_caps->max_input_width            = VP_MAX_PIC_WIDTH;
-        pipeline_caps->max_input_height           = VP_MAX_PIC_HEIGHT;
+        if(mediaCtx->platform.eRenderCoreFamily == IGFX_GEN8_CORE)
```

Comment:
if we will suddenly decide to support something old like HSW, we will fall to the Gen9+ case:(. Well, I understand that this is unlikely, but still catches an eye. What was max resolution for the previous platforms? If they all supported the same, it may be easier to add it here. Just in case...

Don't consider this as a critical comment...

Reply:
For previous platforms, the max supported resolution should be same as Gen8, i.e. 16352.  The driver now only support Gen8+ platform, so we didn't consider previous platform here.